modules/fundamental/src/advisory/service/mod.rs

```diff
@@ -117,7 +117,11 @@ impl AdvisoryService {
                         // v = "" for all sort fields
                         ("average_severity", "") => Some(format!("average_score:{op}")),
                         _ => None,
-                    }),
+                    })
+                    .json_keys(
+                        "labels",
+                        &["type", "source", "importer", "file", "datasetFile"],
```

Comment:
A user can apply any label and value. Will it be possible to e.g. search for `ci=my-build-job` too?

Reply:
Not as a result of merging this PR. That's why I was careful to "relate" #491, because this PR doesn't fully "fix" it. But it does give @JimFuller-RedHat a way to filter SBOM's by type.

Reply:
I don't think we should hardcode any labels like that. If we allow filtering, then by any label.

Reply:
We're already hardcoding the labels. Where do you think I got those from?